templates/pages/user/dashboard.html.twig

```diff
@@ -76,35 +79,77 @@
 			</div>
 			{% endif %}
 		</div>
-		<h3>{{ isPrivateDashboard ? 'mes observations' : 'ses dernières observations' }}</h3>
-		{% set periodsNames = constant('App\\Entity\\Event::DISPLAY_LABELS') %}
-		<div class="dashboard-obs-info-container">
-			{% for observation in observations %}
-				{% set pictureUrl = displayImageToObservation(observation)|default %}
-				<div class="list-cards-item obs">
-					{% if observation.picture is not null %}
-						<a href="{{ pictureUrl }}" class="list-card-img" style="background-image:url({{ pictureUrl }});border:3px dotted #ababab;" title="Image d’observation" target="_blank"></a>
-					{% else %}
-						<div class="list-card-img" style="background-image:url({{ pictureUrl }})"></div>
-					{% endif %}
-					<div class="item-name-block">
-						<div class="item-name">{{ observation.user.displayName }}</div>
-						<div class="item-name stage">{{ periodsNames[observation.event.name] }}{{ observation.event.stadeBbch != null ? " - stade #{observation.event.stadeBbch}" }}</div>
-						<div class="item-heading-dropdown">{{ observation.date|localizeddate('none', 'none', 'fr', null, 'd/MM/y') }}</div>
-					</div>
-					<div class="item-details">
-						<div class="grey">{{ observation.individual.species.vernacularName }}</div>
-						<div>Station :
-							<a href="{{ path("stations_show", {'slug': observation.individual.station.slug}) }}" class="bolder">
-								{{ observation.individual.station.name }}
-							</a>
+
+
+		{% if observations is not empty %}
```

Comment:
I think it could be better for page structure to keep an empty Observations list with a foreach-else "no observations yet" or something. Same for Publications

Reply:
publications creation is not yet public